test/read-file-sync.test.js

```diff
@@ -43,7 +43,7 @@ describe('+ readFileSync()', function () {
         jf.readFileSync(file)
       }, function (err) {
         assert(err instanceof Error)
-        assert(err.message.match(file))
+        assert(err.message.match(/somefile\.json/))
```

Comment:
Ah, I see, the problem is that the string is coerced into a regular expression.

This should work:

```javascript
assert(err.message.indexOf(file) !== -1)
```

Or even prettier, but I don't think it works on <= 0.12

```javascript
assert(err.message.includes(file))
```